src/commands/utils/help.ts

```diff
@@ -9,22 +9,81 @@ export const help: CommandDefinition = {
     name: 'help',
     description: 'Sends a list of available commands to the user',
     category: CommandCategory.UTILS,
-    executor: (msg) => msg.author.createDM()
-        .then(async (dmChannel) => {
+    executor: async (msg) => {
+        const { author } = msg;
+
+        const embed = makeEmbed({
+            title: 'FlyByWire Simulations | Help',
+            description: makeLines([
+                'Would you like to:',
+                '1. Receive a help DM',
+                '2. Receive a link to our docs site with changelogs',
```

Comment:
```suggestion
                '2. Receive a link with the available commands/changelog on our docs site ',
```

Reply:
This doesn't seem correct.

Reply:
@nistei My bad! You are correct.

I have edited the suggestion. Feel free to take you own spin on it. I was asked to stress that the docs site has the commands list AND changelog.